### ISSUE TITLE
Additional changes after merging PR

### ISSUE THREAD

**Issue by raythx98** [AY2021S1-CS2103T-T17-3/tp#68]
There are minor mistakes in PR #36 and #37 which I found out only after merging into the team repository. In this case, which is the better practice?

1. Create another PR just for these changes
2. Commit to the team repo directly
3. Hide these changes in my future PR (e.g. I create a PR that adds a feature, and include a commit which fix these changes)

**Comment by raythx98** [comment]
I will go ahead with 1 and close this issue via PR
